src/components/ListWraper/ListItem/ListItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+//import PropTypes from 'prop-types';
+
+const ListItem = ({id, name, surname, location, age}) => (
+  <tr dataRowToRemove={id}>
+    <td>
+      {name}
+    </td>
+    <td>
+      {surname}
+    </td>
+    <td>
+      {location}
+    </td>
+    <td>
+      {age}
+    </td>
+    <td className="ListWraper__td ListWraper__td--remove">
+      <button className="btn btn-danger btn-sm" dataRowRemove={id}>X</button>
```

Comment:
co robi ten prop dataRowToRemove??

Reply:
jak napisałem pryz <tr> będzie triggerował by usunąć element ze state, ale widzę, że tutaj inaczej to funkcjonuje niż, zatem zrobię z tego komponent